lib/Stats.js

```diff
@@ -498,6 +501,11 @@ class Stats {
 			colors.bold(obj.version);
 			newline();
 		}
+		if(obj.env) {
+			colors.normal("Environment (--env): ");
+			colors.bold(obj.env);
```

Comment:
Updated to use `JSON.stringify`... I did throw in an argument for for spacing. It can certainly be removed, but seems like it might be helpful when passing more complex objects:

`--env.production --env.platform 'web' --env.version 2`
![screen shot 2017-05-22 at 9 43 32 pm](https://cloud.githubusercontent.com/assets/694253/26336629/3c1da6d0-3f39-11e7-8211-f493b45e88f4.png)

Maybe with this use case in mind, it would make sense to print this after `Public Path: ` with a double newline?

 Worth mentioning more "basic" configurations would just be the single line:

`--env production`
![screen shot 2017-05-22 at 10 01 01 pm](https://cloud.githubusercontent.com/assets/694253/26336816/3971409e-3f3a-11e7-8224-da1f8bdb6a06.png)